package.json

```diff
@@ -34,5 +34,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```

Comment:
Kindly and `big.js` and `prop-types` to `dependencies `
![image](https://user-images.githubusercontent.com/31897434/126053832-9f751d4c-8e07-48f2-a570-f9c56de34208.png)